CoreUtilities/src/au/gov/asd/tac/constellation/utilities/file/FileExtensionConstants.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2010-2021 Australian Signals Directorate
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.utilities.file;
+
+/**
+ *
+ * @author aldebaran30701
+ */
+public class FileExtensionConstants {
+    
+    private FileExtensionConstants() {
+        throw new IllegalStateException("Utility class");
+    }
+    
+    public static final String STAR_EXTENSION = ".star";
+    public static final String BACKUP_EXTENSION = ".bak";
+    public static final String STAR_AUTOSAVE_EXTENSION = ".star_auto";
+    
+    public static final String NEBULA_EXTENSION = ".nebula";
```

Comment:
you can probably group these together (nebula also relates to Constellation files

Reply:
it might also be worth adding a comment above each group just to specify what the grouping of extensions is for (e.g. `// Constellation files`, `// image files`)